others/mongodb-101.md

```diff
@@ -0,0 +1,233 @@
+# Week XX Lesson: MongoDB
+
+**What we will learn today?**
+
+* SQL vs NoSQL
+* Why MongoDB?
+* Mongo Shell Basic Commands
+* MongoDB administration
+   * Aggregation
+   * Replication
+   * Sharding
+* What's next?
+
+# Before we start
+
+Download and install MongoDb from
+https://www.mongodb.com/download-center#community . Follow the instructions for
+your platform (Windows, Linux or OS X).
+
+Open two instances of terminal. Run `mongod` in the first instance - this will start MongoDB.
+ 
+In the second instance, first download the sample data script:
+```curl https://gist.githubusercontent.com/agiamas/35b2b954cc942f95709273d3cb9d2cf3/raw/1a0b732594c5f5ae23f638c03519a98364f8b0ff/mongodb_data.js > mongo_data.js```
+
+then import the data we just downloaded using 
+```mongo < mongo_data.js```
+
+Finally, run `mongo` in the second terminal instance.
+
+
+# Lesson 1 - SQL and NoSQL
```

Comment:
Have a feeling that this is more of an "exercise 1" rather than "lesson 1" as it feels like a ~5-15 min section